config/v1/types_infrastructure.go

```diff
@@ -234,6 +241,10 @@ type PlatformStatus struct {
 	// Kubevirt contains settings specific to the kubevirt infrastructure provider.
 	// +optional
 	Kubevirt *KubevirtPlatformStatus `json:"kubevirt,omitempty"`
+
+	// EquinixMetal contains settings specific to the Equinix Metal infrastructure provider.
+	// +optional
+	EquinixMetal *EquinixMetalPlatformStatus `json:"equinixmetal,omitempty"`
```

Comment:
equinixMetal ?

Reply:
Done, I was following the convention of the others, but agree this is preferred.